abstract.tex

```diff
@@ -0,0 +1,79 @@
+    \hline
+  \end{tabular}
+\end{table}
+
+One way to test new fuel cycles, such as one that incorporates pyroprocessing, is by using a fuel cycle simulator. The simulator that will be used in this work to model a pyroprocessing fuel cycle is called \cyclus.
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\section{Cyclus}
+\label{sec:cyclus}
+\cyclus \cite{huff_cyclus_intro_2016} is an open-source nuclear fuel cycle simulation framework that tracks transactions of commodities between facilities. In \cyclus, facilities are represented as agents that can be configured to interact with each other through the central marketplace, and the \cyclus ecosystem is full of pre-built \textit{archetypal} facilities which users can import into a model to simulate a variety of fuel cycle scenarios. \cyclus is designed to be flexible and extensible, allowing users to create custom facilities and commodities to model specific scenarios. The framework also includes a variety of tools for analyzing and visualizing simulation results, making it a powerful tool for researchers and policymakers in the nuclear energy field. Many traditional nuclear fuel cycle facilities have archetypes in \cycamore \cite{Carlsen_cycamore_2014}, which is maintained by the \cyclus developers and users.
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\section{PyRe}
+The standard Separations \cycamore archetype has a user defined efficiency and material streams, but provides little insight into the operations in the reprocessing facility. PyRe fills this gap by allowing users to parameterize sub-processes to keep track of material balance, isotopic buildup, power draw, and other observable values that can be used to detect material diversions \cite{westphal_modeling_2019}.
+
+PyRe is a \cyclus archetype developed by Gregory Westphal that tracks material carefully through the \cyclus Separations archetype. PyRe was made to follow the flow of nuclear material through reprocessing facilities, and to work with data that the IAEA collects so that diversion detection can be more accurately modeled. The PRIDE facility in Korea was modeled to demonstrate PyRe’s material diversion detection capabilities. The data gathered using PyRe can be used to help place detectors in optimal locations to more swiftly detect proliferation risks \cite{westphal_pyre_2018}.
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\section{Purpose}
+
+Recently, the \cyclus framework was updated, and the PyRe archetype is no longer functional. The purpose of this work is to update the archetype so that it can be used in \cyclus simulations once more. To show that the PyRe archetype has been accurately updated, the simulations will be repeated to recreate the results generated in PyRe’s introduction.
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%\section{Acknowledgments}
```

Comment:
We might have to leave an acknowledgment for Katy since she is funding your work, but I'm not sure what it would be.